src/config.rs

```diff
@@ -193,9 +193,11 @@ impl Config {
 
 fn clean_path_string(path_string: &str) -> path::PathBuf {
 	let separator_regex = regex::Regex::new(r"\\|/").unwrap();
+	let trailing_regex = regex::Regex::new(r"(/)+$").unwrap();
```

Comment:
Similar note here, we should also get rid of trailing backslashes. This change will work on Linux because the first `replace_all()` will convert everything to forward slashes but not on Windows.